experimental/packages/opentelemetry-instrumentation-fetch/test/fetch.test.ts

```diff
@@ -593,18 +590,6 @@ describe('fetch', () => {
       assert.ok(init.headers.get('foo') === 'bar');
     });
 
-    it('should pass request object as first parameter to the original function (#2411)', () => {
-      const r = new Request(url);
-      return window.fetch(r).then(
-        () => {
-          assert.ok(true);
-        },
-        (response: Response) => {
-          assert.fail(response.statusText);
-        }
-      );
-    });
```

Comment:
This test was added for #2411. It is basically testing that nothing breaks when passing a `Request` object into `fetch` works. However, that pattern is now ubiquitously used in numerous other tests (e.g. see around L550), so I'm not sure this is still necessary anymore. Also as mentioned about, this test is not very realistically written, as the real `fetch()` would never `reject()` with a `Response` like this.